modules/xmpp/JingleSession.js

```diff
@@ -4,6 +4,7 @@ var SDPDiffer = require("./SDPDiffer");
 var SDPUtil = require("./SDPUtil");
 var SDP = require("./SDP");
 var RTCBrowserType = require("../../service/RTC/RTCBrowserType");
+var RTC = require("../RTC/RTC");
```

Comment:
The RTC interface is accessible trough the APP object (APP.RTC). Do we need that line?